### ISSUE TITLE
warning: could not find UI helper 'GitHub.UI'

### ISSUE THREAD

**Issue by ThomasChowdhury** [git-for-windows/git#4190]

**Comment by rimrul** [comment]
This is probably the same issue as #4165.
please test the latest [snapshot](https://wingit.blob.core.windows.net/files/index.html).

**Comment by dscho** [comment]
@ThomasChowdhury it would be good if you could test the snapshot swiftly. I am planning to release a new Git for Windows version today, and it would be truly regrettable if this regression was not fixed and the only reason for that would be lack of testing _before_ the release.

**Comment by mbs-c** [comment]
@dscho We've run into this issue in our organization, I'll test and report back.

**Comment by mbs-c** [comment]
Test log:

Preparation: I deleted my GitHub Enterprise credentials from the Windows credential manager before starting this test.

```
PS> git --version
git version 2.39.0.windows.1
PS> git pull
warning: could not find UI helper 'GitHub.UI'
Select an authentication method for '<Our GHE instance>':
  1. Web browser (default)
  2. Device code
  3. Personal access token
option (enter for default):^C
# While I got the text prompt here, many of my colleagues did not even see that. For them, git never produced any additional output.
# At this point, I installed the latest snapshot
PS> git --version
git version 2.39.0.windows.1.3.g1396a3d5b9.20221216081425
PS> git pull
# UI windows appears
info: please complete authentication in your browser...
Already up to date.
```

tl;dr: Looks good to me. I look forward to the new version; it'll make my job a lot easier, since we're currently in the middle of a large Perforce → Git migration.

I'll also check whether it works on a PC that never had Git installed later today, but that might take a while.

**Comment by dscho** [comment]
@mbs-c thank you for testing. Would you believe that you're the only one to test any regression fix in preparation for Git for Windows v2.39.0(2)? Users like you make my day. Thank you _so_ much!

**Comment by zwl098** [comment]
[https://juejin.cn/post/7177961317699944509](url)

**Comment by zwl098** [comment]
https://juejin.cn/post/7177961317699944509
